src/main/java/com/synopsys/integration/alert/database/system/SystemStatusUtility.java

```diff
@@ -67,6 +65,12 @@ public Date getStartupTime() {
         return getSystemStatus().getStartupTime();
     }
 
+    private Date createCurrentDateTimestamp() {
```

Comment:
Would this code make sense in DateRange? I think we get the current time using the same code elsewhere in Alert